test/e2e/verify/index.spec.ts

```diff
@@ -0,0 +1,22 @@
+import snapshot from '../../../src/';
+import { Wallet } from '@ethersproject/wallet';
+import { test, expect, describe } from 'vitest';
+import { verify } from '../../../src/verify';
+
+describe('verify', () => {
+  test('should verify a signed message', async () => {
+    const client = new snapshot.Client712();
+    const wallet = Wallet.createRandom();
+    const message = {
+      from: wallet.address,
+      alias: Wallet.createRandom().address
+    };
+
+    const receipt: any = await client.alias(wallet, wallet.address, message);
+    const payload: any = await snapshot.utils.getJSON(receipt.ipfs);
+
+    expect(verify(payload.address, payload.sig, payload.data)).resolves.toBe(
+      true
+    );
+  });
```

Comment:
This will call production every time we trigger these tests.?

Reply:
don't think we should do that

Reply:
This is an E2E tests, with the purpose of testing the whole workflow.

We can either remove this, or use testnet instead ?